content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md

```diff
@@ -368,3 +368,23 @@ E.g. to uninstall Cortex, run:
 ```bash
 helm delete cortex -n mla
 ```
+
+## Upgrade
+
+To incorporate the helm-charts upgrade, follow the below steps:
+
+- **Upgrade Loki to version 2.4.0**: Add the following configuration inside `loki.config` key, under `ingester` label in the Loki's `values.yaml` file:
+  ```yaml
+  wal:
+    dir: /var/loki/wal
+  ```
+- **Upgrade Cortex to version 1.9.0**: Delete the existing `cortex-store-gateway` service by running the below command:
```

Comment:
Can we add some description about the reason that we delete this service, and also link the bug and fix from upstream?